### DISCUSSION TITLE
Quais as libs recomendadas para mock em testes unitários?

### DISCUSSION THREAD

**Discussion by cs-wesleymarcolino** [concretesolutions/concrete_android_community#8] [Geral]
Fala pessoal, blz?

Vou começar um projeto novo aqui e tô começando a escrever os primeiros testes, daí preciso fazer mock de uns objetos. Qual lib vcs recomendam pra fazer isso em Kotlin?

**Comment by Shayanne11** [comment]
Olá, eu conheço as libs de Mockk e Mockito-Kotlin, mas abre uma Issue por favor, para fazer essa comparação de forma mais estruturada e poder documentá-la. Ok?
Use este link para abrir a Issue:
https://github.com/concretesolutions/android-docs-reference/issues/new
